command.py

```diff
@@ -0,0 +1,52 @@
+from .plugin import LspIntelephensePlugin
+from LSP.plugin.core.typing import Optional, TypeVar
+import os
+import shutil
+import sublime
+import sublime_plugin
+
+T = TypeVar("T")
+
+
+def expand_variables(var: T, window: Optional[sublime.Window] = None) -> T:
```

Comment:
You can just use `Any` instead of `T`. :)

Reply:
I guess the next version of LSP for ST 3 will come out first :laughing: